release-notes/2023-01/README.md

```diff
@@ -0,0 +1,157 @@
+# SystemLink Enterprise 2022-12 Release Notes
+
+The 2023-01 release bundle for SystemLink Enterprise has been published to <https://niedge01.jfrog.io>. This update include new feature, bug fixes, and security updates. Work with your account representative to obtain credentials to access these artifacts. If you are not upgrading from the previous release, refer to past release notes to ensure you have addressed all required configuration changes.
+
+## New Features and Behavior changes
+
+- Schedule a Jupyter notebook to run at a specific time or periodically.
+- View detailed information about individual test steps within a test result. Refer to Viewing Test Steps by Result.
+- Access SystemLink navigation and user account settings when using the Jupyter environment.
+
+## Helm Chart Breaking Changes
+
+There are no breaking changes in this release.
+
+## Bugs Fixed
+
+Only customer facing bugs have been included in this list.
+
+[closed-bugs-sle-2023-01.xlsx](closed-bugs-sle-2023-01.xlsx)
+
+## Software Bill of Materials and Notices
+
+[SBOM](https://github.com/ni/install-systemlink-enterprise/tree/2023-01/release-notes/2023-01/sbom)
+
+[Notices](https://github.com/ni/install-systemlink-enterprise/tree/2023-01/release-notes/2023-01/notices)
+
+## Versions
+
+**Top Level Helm Chart:** `systemlink 0.10.0`
+
+**Admin Helm Chart:** `systemlink-admin 0.10.0`
+
+### NI Containers
+
+dashboardsui/20230110.4
+
+dataframeservice-kafka-connect/20221205.2
+
+dataframeservice/20230111.12
+
+dremio-apache/23.0.1
+
+executionsui/20230110.2
+
+filesui/20230111.2
+
+grafana-auth-proxy/20221209.2
+
+grafana-rbac-integrator/20221209.2
+
+helium-fileingestionservices/b4219172c9da9a0c7c25dd4b3ca500fabd2d22c2.1
+
+helium-serviceregistry/50d2e8921bd19f4794ef8f79ba60d089768e5737.1
+
+helium-userservices/cf8c5a4ec9c63dd98585dec2de364cd8d4e91b56.1
+
+helium-webappservices/4772a238370543d2bc31c30f8d018b77da7dafb1.1
```

Comment:
This one is now `niartifacts.jfrog.io/rnd-docker-ci/ni/systemlink/helium-webappservices:03c2246e94932bc07f367b8c66cc16f190abe6a6.1`